index.js

```diff
@@ -0,0 +1,40 @@
+import Book from './module/books.js';
+import { bookForm, books } from './module/intro.js';
+import { DateTime } from './module/luxon.js';
+
+books.forEach(Book.createBook);
+bookForm.onsubmit = (e) => {
+  e.preventDefault();
+  const newBook = Book.addBook(new Book(bookForm.title.value, bookForm.author.value));
+  Book.createBook(newBook);
+};
```

Comment:
- [ ] When a new book is added, there is no interaction to show the user that anything has happened. Please empty the inputs when the "Add" button is clicked.
The image below shows that the Book details are still there even after Adding the book 👇 
![image](https://user-images.githubusercontent.com/46839548/177428092-ba183e96-34b1-4011-8eae-b2644c15b044.png)

- [ ] [OPTIONAL] Optionally, add a success response so the user knows that they have successfully added a book

Reply:
- [ ] The inputs are emptied after the book is added.